cdist/conf/type/__pacman_conf/manifest

```diff
@@ -0,0 +1,111 @@
+#get params
+section=$(cat "$__object/parameter/section")
+key=$(cat "$__object/parameter/key")
+value=$(cat "$__object/parameter/value")
+file=$(cat "$__object/parameter/file" 2>/dev/null || echo "")
+state=$(cat "$__object/parameter/state" 2>/dev/null || echo "present" )
```

Comment:
in case of state you can also use the "default" values:

create parameter/default/state and write "present" into it